onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -517,6 +520,8 @@ def parse_arguments():
                         default=None,
                         help="Manually set the model's layer number")
 
+    FusionOptions.add_arguments(parser)
```

Comment:
>parser [](http://example.com/codeflow?start=32&length=6)

can we append one example of fusion options here?
e.g. --disable_embed_layer_norm